diff --git a/dist/Repo.js b/dist/Repo.js
index abf08426abef0c4d150cdc96e3840fa1b4d9f5ef..f7c2316429b598c6779ac9ffeeb2da9700f900f9 100644
--- a/dist/Repo.js
+++ b/dist/Repo.js
@@ -435,7 +435,7 @@ export class Repo extends EventEmitter {
      * Look up a document by URL and wait for it to be ready.
      */
     async find(id, options = {}) {
-        const { signal } = options;
+        const signal = options.signal ?? AbortSignal.timeout(5000);
         if (signal?.aborted) {
             throw new AbortError();
         }
