diff --git a/dist/keyhive/keyhive.js b/dist/keyhive/keyhive.js
index 5b033d46ea82a051006d5d310b6142cc0289f440..39a9e1e1cf083ed9398ac7e91f9e73cef33f1ec7 100644
--- a/dist/keyhive/keyhive.js
+++ b/dist/keyhive/keyhive.js
@@ -31,7 +31,7 @@ async function bootstrapKeyhive(options) {
     const peerId = peerIdFromSigner(active.signer, options.peerIdSuffix);
     // TODO: Server contact card and PeerId are currently just hardcoded for the demo
-    const serverContactCardJson = '{"Rotate":{"payload":{"old":[73,163,230,244,111,233,153,119,133,211,134,237,111,36,52,131,22,50,54,144,150,45,227,235,128,36,33,217,190,198,55,75],"new":[109,115,204,144,178,114,182,238,113,124,4,139,249,76,220,44,128,104,194,68,187,184,82,241,94,145,104,198,159,122,186,43]},"issuer":[215,244,30,111,15,78,235,218,7,241,63,222,141,131,33,22,234,116,180,208,97,235,210,55,202,209,170,178,98,37,223,159],"signature":[178,64,85,76,51,199,196,151,129,14,191,53,127,191,34,223,97,238,95,109,118,179,152,17,205,188,204,177,116,166,147,231,192,201,48,137,19,214,180,45,108,104,34,8,14,63,115,139,215,142,4,179,233,89,150,218,174,168,107,23,8,109,228,6]}}';
-    const serverPeerIdHardcoded = "1/Qebw9O69oH8T/ejYMhFup0tNBh69I3ytGqsmIl358=";
-    console.warn(`[bootstrapKeyhive] before syncServerFromContactCard`);
+    const serverContactCardJson = '{"Rotate":{"payload":{"old":[33,137,111,238,179,125,3,30,133,139,50,112,178,153,171,132,174,170,136,174,242,17,131,192,37,125,253,108,91,65,84,110],"new":[34,214,236,200,87,89,211,226,207,54,34,177,181,49,195,126,244,174,192,163,6,174,225,45,147,0,82,73,79,78,196,60]},"issuer":[247,9,197,140,25,81,187,131,117,217,148,149,178,185,64,74,187,27,132,103,122,139,165,214,120,158,18,104,148,65,76,25],"signature":[92,213,174,209,83,196,175,199,100,26,241,68,150,220,89,224,87,196,169,73,234,232,74,212,191,3,116,4,138,189,221,44,177,140,139,161,19,139,164,49,224,197,190,25,144,134,90,157,255,86,184,5,184,94,70,127,22,40,43,34,200,182,65,8]}}'
+    const serverPeerIdHardcoded = "9wnFjBlRu4N12ZSVsrlASrsbhGd6i6XWeJ4SaJRBTBk=";
+    console.warn(`[bootstrapKeyhive] before syncServerFromContactCard patched new john key`);
     const syncServer = await syncServerFromContactCard(serverContactCardJson, serverPeerIdHardcoded, keyhive, keyhiveStorage);
     console.warn(`[bootstrapKeyhive] after syncServerFromContactCard`);
